### PR TITLE
Strict Template Migrations

### PR DESCRIPTION
**Pull request by ivanchimeno**
### Description

This pr fixes strict template issues for:

- `/app/admin/audits/audits.component.html`
- `app/exercises/shared/result/result-detail.component.html`
- `app/exercises/shared/result/result.component.t`
- `app/overview/course-exams/course-exams.component.html`
- `app/overview/course-exercises/course-exercises.component.html`
- `webapp/app/overview/courses.component.html`
- `app/overview/submission-result-status.component.html`

### CODE REVIEW

File: src/main/webapp/app/admin/audits/audits.component.html [hunk 1]
```diff
@@ -49,7 +49,7 @@ <h4 jhiTranslate="audits.filter.title">Filter by date</h4>
                     </td>
                     <td>{{ audit.type }}</td>
                     <td>
-                        <span *ngIf="audit.data?.message">{{ audit.data.message }}</span>
+                        <span *ngIf="audit.data?.sessionId">{{ audit.data.sessionId }}</span>
```
Comment (FrankeLukas) [inline src/main/webapp/app/admin/audits/audits.component.html:52]: This causes the extra data to be missing on the audits page, e.g.:
Before:
grafik

After:
grafik

**Review by FrankeLukas — CHANGES_REQUESTED** [review]

File: src/main/webapp/app/admin/audits/audits.component.html (hunk 1 above)
Comment (ivanchimeno) [reply src/main/webapp/app/admin/audits/audits.component.html:52]: Hmm it's weird but the compiler says that "message" doesn't exist. I thought it was referring to sessionId but apparently not

Comment (ivanchimeno) [reply src/main/webapp/app/admin/audits/audits.component.html]: Okay I removed it. Can you check if the table is still the same as before?

**Review by SimonEntholzer — APPROVED** [review]
Tested on TS1, the audit messages are shown again. Code looks good :D

**Review by iozbeyli — APPROVED** [review]
Tested on testing session

**Pull request merged**